extensions/pkg/controller/containerruntime/reconciler.go

```diff
@@ -107,6 +107,10 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return r.delete(r.ctx, cr, cluster)
 	}
 
+	if extensionscontroller.IsFailed(cluster) {
+		return reconcile.Result{}, nil
+	}
+
```

Comment:
Shouldn't this be moved up to line 105?

Reply:
[same question as here](https://github.com/gardener/gardener/pull/2279#discussion_r419915471)